docs/release/updating-to-osg-36.md

```diff
@@ -33,8 +33,8 @@ See the list of services below for any special considerations for the OSG 3.6 up
 
 -   [GridFTP services](#replacing-your-gridftp-service) should be replaced with an installation of XRootD standalone.
 
--   [HTCondor pools and access points](#updating-your-htcondor-hosts) should be updated to OSG 3.6 at your earliest
-    convenience.
+-   [HTCondor pools](#updating-your-htcondor-hosts) and [access points](#updating-your-osg-access-point) should be
+    updated to OSG 3.6 at your earliest convenience.
```

Comment:
> earliest convenience

should we be more urgent here? 3.5 went EOL 9 months ago.

Reply:
Fixed